Packs/Core/ReleaseNotes/3_2_4.md

```diff
@@ -0,0 +1,32 @@
+
+#### Playbooks
+
+##### New: Excessive User Account Lockouts
+
+This playbook addresses the following alerts:
+
+- Excessive user account lockouts
+- Excessive account lockouts on suspicious users
+- Excessive user account lockouts from a suspicious source
+
+The playbook investigates and responds to excessive user account lockout alerts. It gathers information about the alert, enriches relevant host data, and analyzes event patterns. This analysis helps distinguish between benign lockouts and lockouts caused by brute-force or password spray attacks.
+
+**Key Stages**:
```

Comment:
Why "Key stages" and not "Playbook stages" as in the README?

Reply:
Followed the template, will DM you